frontend/packages/dev-console/src/components/pipelines/modals/common/SecretAnnotation.tsx

```diff
@@ -1,16 +1,24 @@
 import * as React from 'react';
 import { TextInputTypes } from '@patternfly/react-core';
 import { InputField, DropdownField } from '@console/shared';
+import { SecretType } from '@console/internal/components/secrets/create-secret';
 import { SecretAnnotationType } from '../../const';
 import './SecretAnnotation.scss';
 
 type SecretAnnotationParam = {
   fieldName: string;
+  secretType: SecretType;
   isReadOnly?: boolean;
 };
 
+const helpText = {
+  [SecretType.dockerconfigjson]: 'The base server url (e.g. https://quay.io/)',
+  [SecretType.basicAuth]: 'The base server url (e.g. https://github.com)',
+  [SecretType.sshAuth]: 'Server hostname without schema or path (e.g. github.com)',
```

Comment:
Localize these.

Reply:
We have PR https://github.com/openshift/console/pull/7143 by @vikram-raj  for pipeline i18n. I would like to keep it there. Depending on whats merged first we should update the other PR, I'm fine to rebase this one if the other is merged first.

Reply:
Now localized